### ISSUE TITLE
About admin User type

### ISSUE THREAD

**Issue by xichen01** [Alluxio/alluxio#14908]
**Is your feature request related to a problem? Please describe.**
Any user who knows the Alluxio Master address and can access the Master can use the alluxio client to manage the Alluxio cluster. This is not safe for Alluxio clusters. 
Some end-users should only have permission to use the Alluxio cluster, and should not have permission to manage Allxuio

**Describe the solution you'd like**
Distinguishing between two user roles can solve this problem:
Alluxio administrative user: an account that is used to manage the Alluxio cluster.
Alluxio service user:  an account that can use the Alluxio service but cannot manage the Alluxio cluster

Does Alluxio have a similar feature or plans to develop this feature?

**Comment by jiacheliu3** [comment]
@madanadit Does this look interesting to you?

**Comment by madanadit** [comment]
@jiacheliu3 we have the super-user concept (the user that starts the master process).
admin operations like the fsadmin cli should already be restricted to that user, is that not the case?

**Comment by xichen01** [comment]
Although the concept of `super-user`, any user in the current alluxio can execute the command `fsadmin report`

`testx` is just a non-superuser in this example, it can execute the `fsadmin report` command.
![image](https://user-images.githubusercontent.com/32928346/151531790-e5202064-d171-4096-8690-a603a151c653.png)

**Comment by HelloHorizon** [comment]
@yyongycy can you take a look?

**Comment by yyongycy** [comment]
Since any user can call fsadmin, there should be no permission check.
@HelloHorizon any contributor from community is interested this feature?

**Comment by xichen01** [comment]
@yyongycy I can refer to HDFS development this feature, permission check at the Master side.
Administrative commands can only be run as the HDFS-superuser in HDFS

such as 
![image](https://user-images.githubusercontent.com/32928346/153166317-e38b2930-3a1a-449c-ad8b-b028444e0420.png)

**Comment by yyongycy** [comment]
How about another super user? Would that work?

**Comment by xichen01** [comment]
`root` is a super user, it can run `fsadmin` commands
![image](https://user-images.githubusercontent.com/32928346/153184503-6edc9a84-a6f6-4b9a-91fc-69d67543dc46.png)

![image](https://user-images.githubusercontent.com/32928346/153184836-c3002590-6090-424b-bb3a-df5aa3874f62.png)

**Comment by yyongycy** [comment]
So any super user can run that command successfully, right? Given that is implemented in Alluxio, it is better to follow the HDFS rule.

**Comment by xichen01** [comment]
Whether to distinguish which commands in Alluxio are administrator commands except `alluxio fsadmin`

`hdfs dfsadmin` is not the only administrator command in HDFS
refer to:
https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HDFSCommands.html#Administration_Commands

**Comment by yyongycy** [comment]
Not sure I understand your question correctly, do you mean "hdfs" has other admin command beside hdfsadmin, and whether alluxio has other admin command?

**Comment by xichen01** [comment]
Yes, there are some other (not all admin commands) admin commands in HDFS besides `dfsadmin` that need to be executed by super user.

such as: `crypto` and `balancer`
![image](https://user-images.githubusercontent.com/32928346/153370296-f534a6b5-894d-42c2-81e0-9211587e1d9e.png)
![image](https://user-images.githubusercontent.com/32928346/153370308-b1838845-aabc-4a03-a990-d183b4842ea6.png)

Are there any commands other than `fsadmin` in Alluxio that only super user can execute?

**Comment by yyongycy** [comment]
I'd suggest doing it step by step, and you can firstly handle this one.

**Comment by xichen01** [comment]
OK, I'll implement `alluxio fsadmin` first.

**Comment by xichen01** [comment]
I am developing the authentication function of fsadmin subcommands, I have encountered some problems.
I currently implement it is to authenticate in the server-side RPC. Alluxio's server does not have a "fsadmin server RPC" to handle all fsadmin subcommand of requests. In fact, fsadmin will call `Blockmaster`, 'FileSystemMaster' 'Jobmaster', and other RPC Server.
For example, `fsadmin report` will call `FileSystemMasterClientServiceHandler#getmountTable` by RPC, however,`fs mount` and some other commands also call the `getMountTable` by RPC, so it is difficult to limit the authentication to the subcommand of fsadmin.
This may be a common problem, it is difficult to limit authentication in a specific subcommand, such as the subcommand of `fsadmin`

**Comment by HelloHorizon** [comment]
@yyongycy can you take a look?
